### ISSUE TITLE
Incorrect behavior if statement. 

### ISSUE THREAD

**Issue by DanteCpp** [wxMaxima-developers/wxmaxima#967]
I found out a very annoying bug in the if statement: 
 
![screenshot from 2018-01-30 14-08-27](https://user-images.githubusercontent.com/6254930/35567936-2056df0c-05c7-11e8-854b-7153f3a4dcf9.png)

**Comment by gunterkoenigsmann** [comment]
It unfortunately is not obvious which program an error message comes from. Actually I had to look up in the sources. This time the error message comes from maxima (https://maxima.sourceforge.net), not from wxMaxima.

The good news is that this error seems to have been resolved in maxima since your Operating system was released - and that there is a nightly build at https://code.launchpad.net/~maxima-developers/+archive/ubuntu/maxima-nightly that contains the bugfix. Don't be afraid: I'm using this nightly build myself for production use and there is a big chance that it is decided that in the next few days the snapshot will be named "release" and officially shipped.

The bad news is that if you want to get it packaged in the release of the operating system you are using you'll have to file a bug report there.
